addon/webextension/selector/callBackground.js

```diff
@@ -0,0 +1,17 @@
+/* globals browser */
```

Comment:
It'll likely be just after we land screenshots in the tree, I'll be able to get back to fixing up the eslint package & getting the rules applied.